term.c

```diff
@@ -1666,7 +1673,10 @@ term_fake_flush(int fd)
             break;
         }
         /* see comment in term_drain */
-        if ( DRAIN_DELAY ) usleep(DRAIN_DELAY);
+        if ( DRAIN_DELAY ) {
+            struct timespec d = {0, DRAIN_DELAY * 1000};
```

Comment:
Isn't it enough to have one timespec d for all invocations?

Reply:
There are only two. I'd also rather not have a global.

Reply:
I could also rewrite to use a compound literal.